src/libstd/sys/redox/rand.rs

```diff
@@ -9,15 +9,19 @@
 // except according to those terms.
 
 use io;
-use libc;
 use rand::Rng;
 
-pub struct OsRng;
+// FIXME: Use rand:
```

Comment:
We have to specify a simple `OsRng` algorithm until the `rand:` process is started. If you have more questions, please ask them here. In the future, `rand:` will be in the kernel to allow the `init` process to execute. (Rust currently uses `OsRng` in every process for the `HashState`)